librandom/exp_randomdev.h

```diff
@@ -39,8 +39,15 @@ namespace librandom
 /*BeginDocumentation
 Name: rdevdict::exponential - exponential random deviate generator
 Description: Generates exponentially distributed random numbers.
+Negative values of lambda are allowed and generate a distribution of negative numbers.
```

Comment:
@luis-rr This line is longer than 79 characters, so it fails our code formatting checks. Could you fix it?

Reply:
Just sent another commit.

Out of curiosity, how did you identify the error? I'm looking at Travis' log and all I can see is:
MSGBLD0220: C/C++ files with formatting errors:
MSGBLD0220: ... librandom/exp_randomdev.cpp
MSGBLD0220: ... librandom/exp_randomdev.h

But not the description of the formatting error itself.